Build broken, romfs fails to build (at least on 8018x)

Hi, I've fetched the latest changes (3b0ca29b0e11694ab84e873453fd339c99027db7) and tried to build it for the 8018x, and I'm getting the following error:

```
make -C fs/romfs romfs.a
make[3]: Entering directory '/home/cocus/source/elks/elks/fs/romfs'
ia16-elf-gcc -ffreestanding -fno-inline -melks -mcmodel=small -msegment-relocation-stuff   -I/home/cocus/source/elks/include -I/home/cocus/source/elks/elks/include -I/home/cocus/source/elks/libc/include -D__KERNEL__ -Wall -Os -Wno-strict-aliasing -fno-delete-null-pointer-checks -Wnull-dereference -c -o romfs.o romfs.c
romfs.c: In function ‘romfs_read’:
romfs.c:36:18: error: ‘current’ undeclared (first use in this function)
   fmemcpyb (buf, current->t_regs.ds, (char *)(int) f->f_pos, i->u.romfs.seg, len);
                  ^~~~~~~
romfs.c:36:18: note: each undeclared identifier is reported only once for each function it appears in
romfs.c: In function ‘romfs_lookup’:
romfs.c:115:53: error: ‘current’ undeclared (first use in this function)
     if (!fmemcmpb ((char *)offset + 3, seg_i, name, current->t_regs.ds, len2)) {
                                                     ^~~~~~~
romfs.c: In function ‘romfs_readlink’:
romfs.c:153:17: error: ‘current’ undeclared (first use in this function)
  fmemcpyb (buf, current->t_regs.ds, 0, inode->u.romfs.seg, count);
                 ^~~~~~~
romfs.c: In function ‘romfs_followlink’:
romfs.c:175:10: error: ‘current’ undeclared (first use in this function)
   pds = &current->t_regs.ds;
          ^~~~~~~
make[3]: *** [../../Makefile-rules:243: romfs.o] Error 1
make[3]: Leaving directory '/home/cocus/source/elks/elks/fs/romfs'
make[2]: *** [Makefile:113: fs/romfs/romfs.a] Error 2
make[2]: Leaving directory '/home/cocus/source/elks/elks'
make[1]: *** [Makefile:73: Image] Error 2
make[1]: Leaving directory '/home/cocus/source/elks/elks'
make: *** [Makefile:13: all] Error 2
```

I've tried `make clean` and `make kclean`, but it always hits this.
I understand that "current" refers to some current-process-handle, but I don't know why it might be broken.

Let me know if I can do any testing to help out.

Thanks!

--- a/elks/arch/i86/drivers/block/ssd-sd.c
+++ b/elks/arch/i86/drivers/block/ssd-sd.c
@@ -18,6 +18,7 @@
 #include <linuxmt/config.h>
 #include <linuxmt/rd.h>
 #include <linuxmt/major.h>
+#include <linuxmt/fs.h>
 #include <linuxmt/kernel.h>
 #include <linuxmt/mm.h>
 #include <linuxmt/errno.h>

--- a/elks/fs/romfs/romfs.c
+++ b/elks/fs/romfs/romfs.c
@@ -4,6 +4,7 @@
 #include <linuxmt/types.h>
 #include <linuxmt/errno.h>
 #include <linuxmt/fs.h>
+#include <linuxmt/sched.h>
 #include <linuxmt/stat.h>
 #include <linuxmt/kernel.h>
 #include <linuxmt/mm.h>

